src/sentry/api/endpoints/project_details.py

```diff
@@ -142,7 +142,18 @@ def validate(self, data):
         try:
             config_str = json.dumps(data)
             validate_sampling_configuration(config_str)
-            self.validate_uniform_sampling_rule(data.get("rules", []))
+
+            # If the feature flag 'organizations:dynamic-sampling-demo' is enabled, we skip the uniform rule validation.
+            # This is useful for product demos, as the user will be able to delete uniform rules.
+            if (
+                features.has(
+                    "organizations:dynamic-sampling-demo",
+                    self.context.get("project", {}).organization,
+                    actor=self.context.get("request", {}).user,
+                )
+                is False
```

Comment:
Do we have a standard here to use either `not x` or `is is False`?

Reply:
I noticed that we usually use `not x`